1/contrib/s2i/run

```diff
@@ -12,6 +12,24 @@ source /usr/local/bin/kube-slave-common.sh
 jenkins_config_dir="/opt/openshift/configuration"
 jenkins_config_path="${jenkins_config_dir}/config.xml"
 
+
+CONTAINER_MEMORY_IN_BYTES=`cat /sys/fs/cgroup/memory/memory.limit_in_bytes`
+echo "Container memory in bytes ${CONTAINER_MEMORY_IN_BYTES}"
+DEFAULT_MEMORY_CEILING=$((2**30-1))
+echo "Default memory ceiling ${DEFAULT_MEMORY_CEILING}"
```

Comment:
i'm also not sure where this value is coming from?  if -Xmx isn't set, doesn't java default the heap size to something like 1/4 of what "free" reports?  (and free is going to report the host memory, not the container memory).